ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -509,6 +509,9 @@ private boolean needToRollOverIndex(IndexEntry entry) {
   /**
    * Finds a key in the index and returns the blob index value associated with it. If not found,
    * returns null
+   * <p/>
```

Comment:
wrongly formatted?

Reply:
I don't think so but since github is flagging it, I'll just remove tit

Reply:
Actually I found other places that use this - so should be ok